<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clock</class>
 <widget class="QMainWindow" name="Clock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clock</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>23</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
background-color: qlineargradient(spread:pad, x1:0.493, y1:0, x2:0.492925, y2:0.972, stop:0 rgba(24, 37, 54, 255), stop:1 rgba(81, 88, 100, 255));

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>651</width>
         <height>221</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>215</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(170, 0, 255, 255), stop:0.00497512 rgba(197, 197, 147, 255), stop:1 rgba(163, 200, 165, 255));

border-color:black;
border-width:3px;
border-style:solid;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_clock">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>3</y>
          <width>641</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Leelawadee</family>
          <pointsize>77</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
border-width:0px;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt;12:34:&lt;/span&gt;&lt;span style=&quot; font-size:48pt;  color:#5d5d5d;&quot;&gt;59&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_date">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>135</y>
          <width>631</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
border-width:0px;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#3a3a3a;&quot;&gt;Lundi 24 d√©cembre 2022&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_saint">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>185</y>
          <width>631</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
border-width:0px;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-style:italic; color:#5a5a5a;&quot;&gt;St Patrick&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_shadow">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>641</width>
          <height>144</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Leelawadee</family>
          <pointsize>77</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
border-width:0px;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt; color:#d9d9d9;&quot;&gt;12:34:&lt;/span&gt;&lt;span style=&quot; font-size:48pt; color:#d9d9d9;&quot;&gt;59&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <zorder>label_shadow</zorder>
       <zorder>label_clock</zorder>
       <zorder>label_date</zorder>
       <zorder>label_saint</zorder>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>270</y>
         <width>87</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#aaaaff;&quot;&gt;by &lt;/span&gt;&lt;span style=&quot; color:#8484c6;&quot;&gt;Xenatronics&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_close">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>7</y>
         <width>36</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="clock.qrc">
         <normaloff>:/icones/icones/standard/white/supprimer(1).png</normaloff>:/icones/icones/standard/white/supprimer(1).png</iconset>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="clock.qrc"/>
 </resources>
 <connections/>
</ui>
